analytics-samples/analytics-sample/src/main/java/com/bytegain/analytics/sample/SampleApp.java

```diff
@@ -28,6 +28,8 @@ public void onCreate() {
     // Initialize a new instance of the Analytics client.
     Analytics.Builder builder =
         new Analytics.Builder(this, ANALYTICS_API_KEY)
+//            .testMode()
+//            .localServerPort(5001)
```

Comment:
Do we want to expose this? Customers will never have a localserver

Reply:
Yeah, but I do this so infrequently that I'll forget.   I'll add a comment that it's for ByteGain only.